recipes/conversation_view/attachment_carousel/attachment_carousel.vue

```diff
@@ -191,8 +167,19 @@ export default {
   },
 
   methods: {
+    mediaComponent (type) {
+      switch (type) {
+        case 'image':
+          return 'dt-image-carousel';
+        default:
+          return 'dt-image-carousel';
+      }
```

Comment:
returns the exact same thing? whats the point?

Reply:
yea i think we discussed to ignore and not show anything for type other than video/image. let me change that.

Reply:
i might have to preprocess the mediaList to remove any type other than image/video.